tox.ini

```
@@ -92,6 +92,41 @@ skip_install = true
 allowlist_externals =
   git
 
+
+[testenv:preview]
+description = preview the docs
+deps =
+  sphinx-autobuild
+  -r{toxinidir}/docs/requirements.txt
+  # FIXME: re-enable the "-r" + "-c" paradigm once the pip bug is fixed.
+  # Ref: https://github.com/pypa/pip/issues/9243
+  # -r{toxinidir}/docs/requirements.in
+  # -c{toxinidir}/docs/requirements.txt
```

Try reusing the values from the previous section like this:
```suggestion
  {[testenv:build-docs]deps}
```